pkgs/development/libraries/vte/default.nix

```diff
@@ -92,8 +92,13 @@ stdenv.mkDerivation rec {
     "-D_b_symbolic_functions=false"
   ];
 
-  # error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
-  NIX_CFLAGS_COMPILE = lib.optionalString stdenv.hostPlatform.isMusl "-Wno-unused-command-line-argument";
+  NIX_CFLAGS_COMPILE = lib.optionals stdenv.hostPlatform.isMusl [
```

Comment:
This variable should be a string, see https://github.com/NixOS/nixpkgs/pull/72074

Reply:
Done. Thanks!